setup.cfg

```
@@ -64,6 +64,7 @@ dev_release =
 
 ; All the extra tools used to help with the development process.
 dev =
+    %(test)s
```

Maybe also add all to dev. in this way, you can just install dev and you know that you are ready to rock :)

```suggestion
    %(all)s
    %(test)s
```

---

I've added this along with the change log.